.vscode/launch.json

```diff
@@ -14,6 +14,7 @@
         "${workspaceFolder}/sampleWorkspace/sampleWorkspace.code-workspace"
       ],
       "sourceMaps": true,
+      "env": { "APOLLO_ENGINE_ENDPOINT": "http://localhost:7096/apollo" },
```

Comment:
This environment variable was introduced to set the studio endpoint to some mock server.

This was previously possible via the (undocumented?) config option
```js
{
  engine: {
    endpoint: "http://localhost:7096/apollo"
  }
}
```

But the environment variable is more flexible to use in tests.